client/my-sites/sidebar/sidebar.jsx

```diff
@@ -680,4 +681,6 @@ function mapStateToProps( state ) {
 }
 
 // TODO: make this pure when sites can be retrieved from the Redux state
-export default connect( mapStateToProps, { setNextLayoutFocus, setLayoutFocus }, null, { pure: false } )( localize( MySitesSidebar ) );
+export default sitesObserver(
+	connect( mapStateToProps, { setNextLayoutFocus, setLayoutFocus }, null, { pure: false } )( localize( MySitesSidebar ) )
```

Comment:
Yes, that make sense even though the connect HOC didn't show up in the React Perf wasted table.